Makefile

```
@@ -43,9 +46,11 @@ MSI_VERSION ?= 0.0.1
 # Right now, the only arch we support is amd64 because Envoy doesn't yet support arm64 on Windows
 # https://github.com/envoyproxy/envoy/issues/17572
 # Once that occurs, we will need to set -arch arm64 and bundle accordingly.
-$(WIN_BIN_EXE): $(GORELEASER)
+$(WIN_BIN_EXE):
 	@echo "--- win-bin ---"
-	@GOOS=windows GOARCH=amd64 $(GORELEASER) build --snapshot --single-target --rm-dist
+    # We can't pass GOOS=windows GOARCH=amd64 as it ends up interpreted by go, not goreleaser.
+    # Hence, we build all targets even if we only need Windows.
+	@go run $(goreleaser) build --snapshot --rm-dist
```

I like being better aligned 😄 

```suggestion
	# We can't pass GOOS=windows GOARCH=amd64 as it ends up interpreted by go, not goreleaser.
	# Hence, we build all targets even if we only need Windows.
	@go run $(goreleaser) build --snapshot --rm-dist
```